src/RiakClient/Models/MapReduce/KeyFilters/And.cs

```diff
@@ -13,64 +15,90 @@
 // KIND, either express or implied.  See the License for the
 // specific language governing permissions and limitations
 // under the License.
-
-using Newtonsoft.Json;
-using System;
-using System.Collections.Generic;
-using System.IO;
-using System.Text;
+// </copyright>
 
 namespace RiakClient.Models.MapReduce.KeyFilters
 {
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Text;
+    using Newtonsoft.Json;
+
     /// <summary>
     /// Joins two or more key-filter operations with a logical AND operation.
     /// </summary>
     internal class And : IRiakKeyFilterToken
     {
-        private readonly Tuple<string, List<IRiakKeyFilterToken>, List<IRiakKeyFilterToken>> _kfDefinition;
+        /// <summary>
+        ///  Collection of key filter definitions.
+        /// </summary>
+        private readonly Tuple<string, List<IRiakKeyFilterToken>, List<IRiakKeyFilterToken>> keyFilterDefinition;
```

Comment:
The default rule is to *not* use an underscore prefix, which I do prefer, even if it is different than `corefx` formatting.

Interesting discussion here: http://blogs.msdn.com/b/sourceanalysis/archive/2008/05/25/a-difference-of-style.aspx